PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 (Put an `X` inside the `[ ]` to denote check mark `[X]`.)
 
 
-- [ ] **If creating a new file :**
-  - [ ] added links to it in the README files ?
-  - [ ] included tests with it ?
-  - [ ] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?
+- [X] **If creating a new file :**
+  - [X] added links to it in the README files ?
+  - [X] included tests with it ?
+  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?
```

Comment:
no need to edit template

Reply:
Hello.
First, thanks for your feedback.
I have question about 'edit template', did you mean that I have to erase that X mark
and leave PULL_REQUEST_TEMPLATE.md as same as original file?
I can reply in 3 mintues. Thanks.

Reply:
[Edit] Revert PULL_REQUEST_TEMPLATE.md